draft-ietf-ice-rfc5245bis.xml

```diff
@@ -3023,49 +2779,31 @@ all media streams using those candidates.
 
 <section anchor="sec-restart" title="ICE Restarts">
 <t>
-An agent MAY restart ICE processing for an existing media stream.
-An ICE restart will cause all previous states, excluding the roles
-of the agents, of ICE processing to be flushed and checks to start anew.
-The only difference between an ICE restart and a brand new media session
-is that during the restart media can continue to be sent to the
-previously validated pair, and that a new media session always requires
-the roles to be determined.
+An agent MAY restart ICE for existing media streams.  An ICE restart
+causes all previous state of the media streams, excluding the roles of
+the agents to be flushed.  The only difference between an ICE restart
+and a brand new media session is that during the restart media can
+continue to be sent, and that a new media session always requires the
+roles to be determined.
 </t>
 
-<t>An agent MUST restart ICE for a media stream if:</t>
+<t>If an agent wants to change the destinations of media streams, it
+MAY restart ICE to do so.</t>
 
-<t>
-<list style="symbols">
-<t>
-The candidate(s) is being generated for the purposes of changing the
-target of the media stream.  In other words, if an agent wants to
-generate an updated candidate information that, had ICE not been in
-use, would result in a new value for the destination of a media
-component.
-</t>
-
-<t>
-An agent is changing its implementation level.  This typically
-only happens in third party call control use cases, where the entity
-performing the signaling is not the entity receiving the media, and it
-has changed the target of media mid-session to another entity that has
-a different ICE implementation.
-</t>
-</list>
-</t>
+<t>If an agent want to change its implementation level, it MAY restart
```

Comment:
IETF99: Instead of implementation level say explicitly "ICE lite" and "Full ICE"